index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My_Microverse_Portfolio_Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <h1>Welcome</h1>
+      <button class="btn-trigger">
+        <img src="./assets/img/union.svg" alt="button"/>
+      </button>
+    </header>
+    <section class="headline">
+      <h2>
+        <span>Hey there. I’m Chimwemwe</span>
+        <span class="green-txt">I’m a software developer</span>
+      </h2>
```

Comment:
- [ ] This is good adding `<hx>` tag for the main title, but you don't actually need to have the `<span>` inside of the `<h2>` tag, you can use instead the `<br/>` tag to have the second text on a new line like this 👇 
```
<h2>
     Hey there. I’m Chimwemwe <br/>
     I’m a software developer
</h2>
```
This is going to keep our code simple, clean and more readable 😉